contracts/core/modules/RebalancingSetExchangeIssuanceModule.sol

```diff
@@ -367,4 +370,26 @@ contract RebalancingSetExchangeIssuanceModule is
             "RebalancingSetExchangeIssuanceModule.validateRedeemInputs: Base Set quantities must match"
         );
     }
+    /**
+     * Withdraw any non-exchanged components to the user
+     *
+     * @param  _setAddress   Address of the Base Set
+     */
+    function returnExcessBaseSetComponents(
+        address _setAddress
+    )
+        private
+    {
+        address[] memory baseSetComponents = ISetToken(_setAddress).getComponents();
+        for (uint256 i = 0; i < baseSetComponents.length; i++) {
```

Comment:
Why do we do individual transfers here when we batch them in the `ExchangeIssuanceModule` contract?

Reply:
Nvm we don't have a batch transfer functionality